tests/testDB.js

```diff
@@ -107,7 +73,7 @@ describe('Database Tests', () => {
       assert.isAtMost(results[0].accuracy, 5);
       done();
     });
-  }).timeout(3000);
+  }).timeout(6000);
```

Comment:
What's this timeout for? Could this be better handled with a callback or promise?